views/pages/index.ejs

```diff
@@ -1,12 +1,29 @@
-<%- include("../partials/header.ejs", {title: "Main Page"}) %>
 
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    
+</head>
+<body>
+    <nav>
+        <ul>
+            <li><a href="/fullEpisodes">Full Episodes</a></li>
+            <li><a href="/Contact">Contact Us</a></li>
+        </ul>
+    </nav>
+    <h1>Welcome to the Game of Thrones World!</h1>
+    
+    <section>
+        <p><a href="/db-test">Clicking this link will test to see if your db connection is ok</a></p>
+      </section>
```

Comment:
you've removed the request-handler for the path /db-test from app.js so you should remove this link from your html page too so that no one clicks it and gets a 404 error